gems/smithy/lib/smithy/views/client.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Views
+    # @api private
+    module Client; end
+  end
+end
+
+# helper classes
+
```

Comment:
Style nit: extra line?

Reply:
I'll change it but I prefer attaching docstrings to code if it's for the code and I use new lines for separating sections.